salt/utils/winapi.py

```diff
@@ -4,11 +4,32 @@
 
 # Import python libs
 import logging
-import pythoncom
 import threading
+from salt.loader import minion_mods
+
+try:
+    import pythoncom
+    HAS_LIBS = True
+except ImportError:
+    HAS_LIBS = False
 
 log = logging.getLogger(__name__)
 
+__salt__ = None
+
+
+def __virtual__():
+    '''
+    Only load if required libraries exist
+    '''
+    if not HAS_LIBS:
+        return False
+    else:
+        global __salt__
+        if not __salt__:
+            __salt__ = minion_mods(__opts__)
```

Comment:
This is going to slow down a lot of operations in Salt by 2x. I'm a little surprised that `__salt__` isn't being injected here. We need to investigate this one further.